sdks/typescript/src/apache_beam/runners/dataflow.ts

```diff
@@ -33,6 +33,8 @@ export function dataflowRunner(runnerOptions: {
       options: Object = {}
     ): Promise<PipelineResult> {
       var augmentedOptions = { experiments: [] as string[], ...options };
+      augmentedOptions.experiments.push("use_runner_v2");
+      augmentedOptions.experiments.push("use_portable_job_submission");
```

Comment:
Would typescript tests have caught this? If yes, should we add python runner paths to the workflow trigger so that the tests run against python dataflow runner changes?

Reply:
Yes, the TypeScript Dataflow Tests should have caught this, but it's not generally run due to lack of credentials.

Reply:
> but it's not generally run due to lack of credentials.

Not sure what this means - https://github.com/apache/beam/actions/workflows/typescript_tests.yml looks like its running consistently